src/main/java/no/fintlabs/AzureClient.java

```diff
@@ -26,10 +26,11 @@ public class AzureClient {
     private final ResourceGroupMembershipConsumerService resourceGroupMembershipConsumerService;
 
     private void pageThrough(AzureGroup azureGroup, DirectoryObjectCollectionWithReferencesPage inPage) {
+        int members = 0;
         log.info("Fetching Azure Groups");
         DirectoryObjectCollectionWithReferencesPage page = inPage;
         do {
-            log.info("Membership detected");
+            members++;
```

Comment:
Skulle members vært inne i for loopen? Nå vil den alltid bli minst 1 fordi verdien økes før start.